### ISSUE TITLE
Performance of Pre-Trained DETR

### ISSUE THREAD

**Issue by ShunchiZhang** [ubc-vision/IterativeSG#3]
Hi Dr. Khandelwal @siddheshk,

Did you test the performance (e.g.: Average Precision) of your pre-trained DETR on Visual Genome dataset? Or would you mind giving me some instructions on how to do so using your codebase?

Thanks for taking your time!

**Comment by siddheshk** [comment]
I think this was the performance of the pre-trained DETR on Visual Genome detection.

|   AP   |  AP50  |  AP75  |  APs  |  APm   |  APl   |
|:------:|:------:|:------:|:-----:|:------:|:------:|
| 17.303 | 31.122 | 16.506 | 5.511 | 13.066 | 23.519 |


You can train DETR if needed using this codebase. You will need to use the following flags when running the training code.
`MODEL.META_ARCHITECTURE 'Detr' MODEL.DETR.NAME 'DETR'  MODEL.DETR.TRANSFORMER 'Transformer' MODEL.DETR.CRITERION 'SetCriterion' MODEL.DETR.MATCHER 'HungarianMatcher'  SOLVER.IMS_PER_BATCH '16' SOLVER.STEPS '(369600,)' SOLVER.MAX_ITER '554400'`
The learning rate and steps are taken from the original DETR implementation, which can be found [here](https://github.com/facebookresearch/detr).
